pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.black]
```

Comment:
What's the purpose of this file?

Reply:
It is a configuration for black to override black default values for lining and code styling.